dbt/adapters/synapse/__init__.py

```diff
@@ -10,5 +10,5 @@
     adapter=SynapseAdapter,
     credentials=SynapseCredentials,
     include_path=synapse.PACKAGE_PATH,
-    dependencies=['sqlserver'])
+    dependencies=['sqlserver']
```

Comment:
is this enough to have `dbt-sqlserver`'s `sqlserver__` macros made available?

Reply:
Right, this is what enables you do things like:
```
{% macro synapse__get_columns_in_query(select_sql) %}
  {{ return(sqlserver__get_columns_in_query(select_sql)) }}
{% endmacro %}
```

After we do fishtown-analytics/dbt#2923, dbt will know to look for `synapse__`, then `sqlserver__`, then `default__`, so it should _just work_ without the explicit reimplementation.